src/components/ProgressBar/index.js

```diff
@@ -19,15 +20,15 @@ export default function ProgressBar(props) {
      } = props;
 
     const getContainerClassNames = () => classnames(
-            'slds-progress-bar',
-            `slds-progress-bar_${size}`,
-            { 'slds-progress-bar_circular': variant === 'circular' },
+            'rainbow-progress-bar',
+            `rainbow-progress-bar_${size}`,
+            { 'rainbow-progress-bar_success': color === 'success', 'rainbow-progress-bar_circular': variant === 'circular' },
```

Comment:
remove color prop and use variant to be 'brand' or 'success' since always it will be circular